libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -183,11 +196,20 @@ bool ASTJsonConverter::visit(FunctionDefinition const& _node)
 
 bool ASTJsonConverter::visit(VariableDeclaration const& _node)
 {
-	addJsonNode(_node, "VariableDeclaration", {
-		make_pair("name", _node.name()),
-		make_pair("type", type(_node))
-	}, true);
+	std::vector<pair<string const, Json::Value const>> attributes = {
+                make_pair("name", _node.name()),
+		make_pair("type", type(_node)),
+		make_pair("constant", _node.isConstant()),
+		make_pair("storageLocation", location(_node.referenceLocation())),
+		make_pair("visibility", visibility(_node.visibility()))
+        };
+	if (inEvent)
+          {
```

Comment:
Please do not use curly braces for single statements. If you have to, please as follows:
```
if (inEvent)
{
    ...
}
```